Makefile

```diff
@@ -0,0 +1,165 @@
+covargs := --append --source=src/pudl
```

Comment:
We could add to more targets:

- A target to launch dagster webserver with some default arguments `dagster-webserver -m pudl.etl -m pudl.ferc_to_sqlite`
- A target to build the mamba environment and install pudl

Reply:
Yeah, I think this will be a nice place to put commands that we're all kind of doing in our own ways repeatedly and standardize them a bit.